app/src/main/res/values/strings.xml

```diff
@@ -50,8 +50,8 @@
     <string name="SettingsActivity.Title.Edit">Edit</string>
     <string name="SettingsActivity.Title.Save">Save</string>
     <string name="SettingsActivity.Title.EnterIntranetUrl">Enter your intranet URL</string>
-    <string name="SettingsActivity.Title.DeleteConfirmation">Delete server</string>
-    <string name="SettingsActivity.Message.DeleteConfirmation">Do you really want to delete this server ?</string>
+    <string name="SettingsActivity.Title.DeleteConfirmation">Delete intranet</string>
+    <string name="SettingsActivity.Message.DeleteConfirmation">Do you really want to delete this eXo ?</string>
```

Comment:
```suggestion
    <string name="SettingsActivity.Message.DeleteConfirmation">Do you really want to delete this eXo instance ?</string>
```

Reply:
@ahamdi @hghorbel Yes i guess is better like that.